OpenSim/Simulation/SimbodyEngine/Test/testJoints.cpp

```diff
@@ -2226,6 +2231,42 @@ void testAutomaticJointReversal()
     ASSERT_EQUAL(accErr, 0.0, sqrt(integ_accuracy));
 }
 
+void testUserJointReversal()
+{
+    using namespace OpenSim;
+
+    cout << "\n==========================================================="
+         << "\n Test joint reversal set by user in old model file"
+         << "\n==========================================================="
+         << endl;
+
+    // Open model.
+    auto model = Model("double_pendulum_testReverse.osim");
+    //model.finalizeFromProperties();
+    model.initSystem();
```

Comment:
The throw occurs in [Socket::getConnectee()](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Common/ComponentSocket.h#L329). The comment for this method begins, "Temporary access to the connectee for testing purposes." so I don't know why this method is being called at all or whether it should still exist at this point.